app.js

```diff
@@ -1,32 +1,55 @@
-const express = require('express');
-const bodyParser = require('body-parser');
+const express = require("express");
+const bodyParser = require("body-parser");
 
 // Import controllers
-const createPatronV0_1 = require('./api/controllers/v0.1/createPatron.js'); // eslint-disable-line camelcase
-const validations = require('./api/controllers/v0.1/validations.js');
-const createPatronV0_2 = require('./api/controllers/v0.2/createPatron.js'); // eslint-disable-line camelcase
-const apiDoc = require('./api/controllers/apiDoc.js');
-const createPatronV0_3 = require('./api/controllers/v0.3/endpoints.js'); // eslint-disable-line camelcase
+const createPatronV0_1 = require("./api/controllers/v0.1/createPatron.js"); // eslint-disable-line camelcase
+const validations = require("./api/controllers/v0.1/validations.js");
+const createPatronV0_2 = require("./api/controllers/v0.2/createPatron.js"); // eslint-disable-line camelcase
+const apiDoc = require("./api/controllers/apiDoc.js");
+const createPatronV0_3 = require("./api/controllers/v0.3/endpoints.js"); // eslint-disable-line camelcase
+
+const awsDecrypt = require("./config/awsDecrypt");
 
 const app = express();
-const pathName = `${process.cwd()}/config/deploy_${app.get('env')}.env`;
-require('dotenv').config({ path: pathName });
-
-const BarcodeDb = require('./db');
-
-// Initialize the connection to the database.
-// This is done here so we can have one instance and one connection to the
-// database for however many times the lambda is used for.
-const db = BarcodeDb({
-  user: process.env.DB_USER,
-  host: process.env.DB_HOST,
-  database: process.env.DB_DATABASE,
-  password: process.env.DB_PASSWORD,
-  port: process.env.DB_PORT,
-});
-// Once we get the database class instance, initialize it by creating the
-// table and inserting the seed data if it's not in the database already.
-db.init();
+const pathName = `${process.cwd()}/config/deploy_${app.get("env")}.env`;
+require("dotenv").config({ path: pathName });
+
+const BarcodeDb = require("./db");
+
+// Decrypt any database credentials if on QA or production for the NYPL AWS
+// environment. Then create the single database instance in the app.
+async function initDatabase() {
```

Comment:
This is the new part, everything else is mostly eslint/prettier doing its thing.